core/frontend/src/render/webgl/glsl/FeatureSymbology.ts

```diff
@@ -125,9 +129,11 @@ float computeLineCode() {
 
 function addFeatureIndex(vert: VertexShaderBuilder): void {
   vert.addGlobal("g_featureIndex", VariableType.Vec3);
-  vert.addFunction(getFeatureIndex(vert.usesInstancedGeometry));
 
-  if (!vert.usesInstancedGeometry)
+  vert.addFunction(decodeUint24);
+  vert.addFunction(getFeatureIndex(vert));
+
+  if (vert.usesVertexTable)

```

Comment:
This used to only execute for uninstanced geometry - was it intentionally changed to execute for instanced geometry too?

Reply:
Definitely an error.  Fixed.